cmd/agent.go

```diff
@@ -469,7 +469,7 @@ var agentListCmd = &cobra.Command{
 var agentGetApiKeyCmd = &cobra.Command{
 	Use:     "apikey [agent_name]",
 	Short:   "Get the API key for an agent",
-	Args:    cobra.MaximumNArgs(1),
+	Args:    cobra.ExactArgs(1),
```

Comment:
If it's going to be exactly one arg, there's no need to check for `len(args)` in line 490